components/playback/controls/PlateBarcode.vue

```diff
@@ -6,69 +6,49 @@
     <!-- original Mockflow ID: cmpDd0be63536ca605546f566539e51ad0c3-->
     <input
       id="plateinfo"
-      :disabled="
-        playback_state === playback_state_enums.RECORDING ||
-        playback_state === playback_state_enums.BUFFERING ||
-        playback_state === playback_state_enums.LIVE_VIEW_ACTIVE
-      "
+      disabled="disabled"
```

Comment:
Please add an E2E test attempting to type in the textbox to confirm that the user cannot type

Reply:
Added a new testcase now attempting to type in the textbox with a screenshot `basic-entry-failure-base.png` captures the same.